packages/cli/generators/app/templates/src/controllers/ping.controller.ts.ejs

```diff
@@ -1,5 +1,32 @@
 import {Request, RestBindings, get} from '@loopback/rest';
 import {inject} from '@loopback/context';
+import {ResponseObject} from '@loopback/openapi-v3-types';
```

Comment:
Can we import `ResponseObject` from `@loopback/rest` directly? I believe our rest package is re-exporting OAS types for ease of use.

We are going to remove `@loopback/openapi-v3-types` soon (before 4.0 GA), see https://github.com/strongloop/loopback-next/issues/301.

Reply:
I was concerned about https://github.com/Microsoft/TypeScript/issues/26985. Will double-check.

Reply:
I add the code to re-export `@loopback-v3-types` from `@loopback-rest`.